membership-attribute-service/app/components/TouchpointComponents.scala

```diff
@@ -143,20 +143,20 @@ class TouchpointComponents(
   )
 
   lazy val futureCatalog: Future[CatalogMap] = catalogService.catalog
-    .map(_.fold[CatalogMap](error => { println(s"error: ${error.list.toList.mkString}"); Map() }, _.map))
+    .map(_.fold[CatalogMap](error => { logger.errorNoPrefix(scrub"error: ${error.list.toList.mkString}"); Map() }, _.map))
     .recover { case error =>
-      logger.error(scrub"Failed to load the product catalog from Zuora due to: $error")
+      logger.errorNoPrefix(scrub"Failed to load the product catalog from Zuora due to: $error")
       throw error
     }
 
   lazy val subscriptionService: SubscriptionService = {
-    lazy val zuoraSubscriptionService = new ZuoraSubscriptionService(productIds, futureCatalog, zuoraRestClient, zuoraSoapService.getAccountIds)
+    lazy val zuoraSubscriptionService = new ZuoraSubscriptionService(productIds, futureCatalog, zuoraRestClient, zuoraSoapService)
```

Comment:
this wasn't trivial to get to compile with the new implicit parameter.  Rather than making a suitable trait, I just passed in the whole service.  It may be excessive to give it the whole thing, but there were no tests relying on it so it didn't cause any extra code there.